lighty-tests/test-report-aggregator/pom.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.lighty.core</groupId>
+        <artifactId>lighty-parent</artifactId>
+        <version>15.0.0-SNAPSHOT</version>
+        <relativePath>../../lighty-core/lighty-parent</relativePath>
+    </parent>
+
+    <groupId>io.lighty.tests.report.aggregator</groupId>
+    <artifactId>test-report-aggregator</artifactId>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+    </properties>
+
+    <!-- For the correct result of lighty coverage, is necessary to add here dependencies with
+     IT tests or testable code from lighty.  -->
+
+    <dependencies>
+        <!--lighty applications -->
+        <dependency>
+            <groupId>io.lighty.applications.rnc</groupId>
+            <artifactId>lighty-rnc-app</artifactId>
+            <version>15.0.0-SNAPSHOT</version>
```

Comment:
versions should be controlled via dependency management in BOM pom or sufficient parent's pom file

Reply:
done